### PR TITLE
implement individual project intro page

### PR DESCRIPTION
**Pull request by erizocosmico**
screencapture-localhost-8181-intro-1500472417598

*Notes:*

- The codebox will be done in a followup PR.
- The navbar logo is small because I didn't have the white one and the ratio of that one is not correct.

### CODE REVIEW

File: hugo/content/project/documentation.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+type: project/get-started
+classes: ["static"]
+section: project-get-started
+url: get-started
+title: Job opportunities
+description: Our Job Opportunities. Exceptional missions require exceptional people
```
Comment (Serabe) [inline hugo/content/project/documentation.md]: what's this?

Comment (erizocosmico) [inline hugo/content/project/documentation.md]: the get started project page that is yet a todo

**Comment by erizocosmico** [comment]
PR has been updated with the correct design. PTAL @Serabe @dpordomingo

**Comment by dpordomingo** [comment]
I think the navbar logo should be hosted by each project (bblfsh repo, in that example)

**Comment by erizocosmico** [comment]
Yeah, it's in the project yaml, it can either be a remote URL or a local one

**Comment by erizocosmico** [comment]
Pushed with the styles of the content:

screencapture-localhost-8181-intro-1500476615590

**Comment by Serabe** [comment]
Looks beautiful. Thanks!

**Comment by dpordomingo** [comment]
ping @marnovo

**Pull request merged**